great_expectations/core/expectation_suite.py

```diff
@@ -141,6 +143,56 @@ def expectations(self) -> list[Expectation]:
             for expectation_configuration in self.expectation_configurations
         ]
 
+    def add(self, expectation: Expectation) -> Expectation:
+        """Add an Expectation to the collection."""
+        if not any(
+            expectation.configuration == existing_config
+            for existing_config in self.expectation_configurations
+        ):
+            self.expectation_configurations.append(expectation.configuration)
+        else:
+            pass  # suite is a set-like collection
+        try:
+            self.save()
+        except Exception as exc:
+            # rollback this change
+            self.expectation_configurations.pop()
+            raise exc
+
+        return expectation
+
+    def delete(self, expectation: Expectation) -> Expectation:
+        """Delete an Expectation from the collection.
+
+        Raises:
+            ValueError: Expectation not found in suite.
+        """
+        updated_expectation_configs = [
```

Comment:
nit: the name here initially made me think that the expectation configs (rather than the list) were updated. I'd probably go with `remain_expectation_configs`

Reply:
sure, updated in [0f46abf](https://github.com/great-expectations/great_expectations/pull/8986/commits/0f46abf11088632a96dcf2ad12bef1b1a6d64d66)